src/account_multisig.ts

```diff
@@ -20,14 +20,36 @@ export const MULTISIG_DEPOSIT = new BN('0');
 export const MULTISIG_CHANGE_METHODS = ['add_request', 'add_request_and_confirm', 'delete_request', 'confirm'];
 export const MULTISIG_VIEW_METHODS = ['get_request_nonce', 'list_request_ids'];
 export const MULTISIG_CONFIRM_METHODS = ['confirm'];
+export const MULTISIG_CONTRACT_HASHES = process.env.MULTISIG_CONTRACT_HASHES || [
```

Comment:
@mattlockyer probably need to parse `process.env.MULTISIG_CONTRACT_HASHES`? Though I'd prefer we don't rely on env variables in near-api-js (stuff should be passed through config).

Reply:
removed